lib/common/json-schema-validator.js

```diff
@@ -40,6 +42,9 @@ function jsonSchemaValidatorFactory(
      * @return {Boolean} validation pass or throw error
      */
     JsonSchemaValidator.prototype.validate = function (schema, data) {
+
+        sanitizer.decrypt(data);
```

Comment:
The `decrypt` should not be put in json schema validator. The validator is intented to validate whether the data conforms to the schema, it will not modify the data itself. So please find another reasonable place to put such logic.